enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -15,15 +20,25 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
 	"github.com/buildbuddy-io/buildbuddy/server/util/alert"
 	"github.com/buildbuddy-io/buildbuddy/server/util/background"
+	"github.com/buildbuddy-io/buildbuddy/server/util/disk"
 	"github.com/buildbuddy-io/buildbuddy/server/util/log"
 	"github.com/buildbuddy-io/buildbuddy/server/util/networking"
 	"github.com/buildbuddy-io/buildbuddy/server/util/random"
+	"github.com/buildbuddy-io/buildbuddy/server/util/retry"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 )
 
 var (
+	// Note: to get these cgroup paths, run a podman container like
+	//     podman run --rm --name sleepy busybox sleep infinity
+	// then look at the output of
+	//     find /sys/fs/cgroup | grep libpod-$(podman container inspect sleepy | jq -r '.[0].Id')
+
+	memUsagePathTemplate = flag.String("executor.podman.memory_usage_path_template", "/sys/fs/cgroup/memory/libpod_parent/libpod-{{.ContainerID}}/memory.usage_in_bytes", "Go template specifying a path pointing to a container's current memory usage, in bytes. Templated with `ContainerID`.")
+	cpuUsagePathTemplate = flag.String("executor.podman.cpu_usage_path_template", "/sys/fs/cgroup/cpuacct/libpod_parent/libpod-{{.ContainerID}}/cpuacct.usage", "Go template specifying a path pointing to a container's total CPU usage, in CPU nanoseconds. Templated with `ContainerID`.")
+
```

Comment:
Could we have a setup function that determines these paths when podman first runs (sync.Once style)?

I get a little scared of these super longs paths like this that need to be specified and what happens when they don't work on some on-prem system that is different than ours.

Or if you are pretty sure they will never change, I'm happy to roll with this.

Reply:
I wasn't super thrilled with these either. I thought about getting these directly from podman, but podman doesn't expose the cgroup paths via any of the CLI commands as far as I can tell (e.g. `podman container inspect`).

One thing I can think to do would be to run a small container like busybox, and then run the Go-equivalent of a command like

`find /sys/fs/cgroup | grep libpod-${CONTAINER_ID} | grep -P '(/cpu.usage|/memory.usage_in_bytes|/memory.current)'`

Then derive the cgroup paths from those paths. I'm not sure it's worth the complexity so I figured we could hard code for now then maybe do something smarter later on -- wdyt?

Reply:
sgtm